pkg/kubelet/server/stats/volume_stat_calculator_test.go

```diff
@@ -79,13 +89,15 @@ func TestPVCRef(t *testing.T) {
 	mockStats := new(statstest.StatsProvider)
 	volumes := map[string]volume.Volume{vol0: &fakeVolume{}, vol1: &fakeVolume{}}
 	mockStats.On("ListVolumesForPod", fakePod.UID).Return(volumes, true)
+	blockVolumes := map[string]volume.BlockVolume{vol2: &fakeBlockVolume{}}
+	mockStats.On("ListBlockVolumesForPod", fakePod.UID).Return(blockVolumes, true)
```

Comment:
Based on the above comment, what happens to this test if this method returned false for the second value?

Reply:
In that case, `blockVolumes` is expected to be an empty map too. Stats for `vol2` should not get gathered.